docs/helm-broker/docs/03-04-bundles-docs.md

```diff
@@ -7,7 +7,7 @@ Using the Helm Broker, you can provision a bundle which provides its own Service
 
 ```
 apiVersion: cms.kyma-project.io/v1alpha1
-kind: ClusterDocsTopic
```

Comment:
user can define both the ClusterDocsTopic and DocsTopic, so you can have two kinds in one examples. Please also fix the documentation about it. Right now we are mentioning only the ClusterDocsTopic. 

Maybe we can also mention that the name of the file and place where it is stored is up to the user. It does not have to be called `docs.yaml`